package.json

```diff
@@ -5,10 +5,15 @@
   "devDependencies": {
     "@babel/preset-env": "^7.1.6",
     "lerna": "^3.4.3",
-    "prettier": "^1.15.2"
+    "prettier": "^1.15.3",
+    "prettier-plugin-solidity": "^1.0.0-alpha.12"
   },
   "scripts": {
-    "test": "npx lerna run test"
+    "bootstrap": "npx lerna bootstrap",
+    "ganache:start": "npm run ganache:stop && npx lerna run ganache:start --scope tasit-contracts",
```

Comment:
So the `ganache:start` run by lerna is the npm script in the `tasit-contracts` package, right?